merlin-sdk/src/main/java/gov/nasa/jpl/aerie/merlin/protocol/types/Duration.java

```diff
@@ -533,25 +599,21 @@ public boolean isPositive() {
    */
 
   public boolean isZero() {
-    return this.durationInMicroseconds == 0;
+    return this.micros == 0;
   }
 
   public boolean isEqualTo(final Duration other) {
     if(other == null) return false;
-    return this.durationInMicroseconds == other.durationInMicroseconds;
+    return this.micros == other.micros;
   }
 
-  /** Determine whether two durations are the same. */
-  @Override
-  @Deprecated
-  public boolean equals(final Object o) {
-    if (!(o instanceof final Duration other)) return false;
-    return this.isEqualTo(other);
```

Comment:
I think `isEqualTo` original existed to give some type safety, so both operands would have to be `Duration`, and only the right operand could be `null`. Personally I think this was a little weird, because the answer to "does my `Duration` equal my `String`" is a perfectly sound "no".

I'd prefer to delete this function and replace its usages with `.equals`, which is of course the standard. But I can also just have `isEqualTo` delegate to `.equals` if you want.

Reply:
What I'm proposing is to replace its usage with `.equals` but to override `.equals` to have the _body_ of `isEqualTo`, since our code currently expects the logic of `isEqualTo`.

So
```java
@Override
public boolean equals(final Object o) {
// (null check from `isEqualTo` is covered by `instanceof` check)
if (!(o instanceof final Duration other)) return false;
return this.micros == other.micros;
```

Reply:
Yep, that is pretty much verbatim what the default implementation is for records.